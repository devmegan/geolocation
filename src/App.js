import Nav from "./component/Nav";
import Hero from "./component/Hero";

export default function App() {
  return (
    <div data-theme="bumblebee">
      <Nav />
      <Hero />
    </div>
  )
}