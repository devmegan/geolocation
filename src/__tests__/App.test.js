import { render } from '@testing-library/react';
import App from '../App';

describe('app', () => {
  it('renders w/o crashing', () => {
    expect.hasAssertions();

    const view = render(<App />);
    
    expect(view).toMatchSnapshot();
  });
})
