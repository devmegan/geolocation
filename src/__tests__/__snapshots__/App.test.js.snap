// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`app renders w/o crashing 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        data-theme="bumblebee"
      >
        <nav
          class="navbar bg-primary"
        >
          <div
            class="flex-1"
          >
            <a
              class="btn btn-ghost normal-case text-xl"
              href="/"
            >
              useGeolocation
            </a>
          </div>
          <div
            class="flex-none"
          >
            <ul
              class="menu menu-horizontal p-0"
            >
              <li>
                <a
                  href="https://github.com/streamich/react-use/blob/master/docs/useGeolocation.md"
                  target="__blank"
                >
                  Docs
                </a>
              </li>
            </ul>
          </div>
        </nav>
        <div
          class="hero min-h-screen bg-base-200"
        >
          <div
            class="hero-content text-center"
          >
            <div
              class="max-w-md"
            >
              <h1
                class="text-5xl font-bold"
              >
                useGeolocation
              </h1>
              <p
                class="py-6"
              >
                A React hook that grabs a user's coordinates.
              </p>
              <button
                class="btn btn-primary btn-wide"
              >
                Get Location
              </button>
              <div
                class="w-1/2 mx-auto mt-4"
              >
                <label
                  class="label cursor-pointer"
                >
                  <span
                    class="label-text"
                  >
                    High accuracy
                  </span>
                  <input
                    class="toggle toggle-primary"
                    type="checkbox"
                  />
                </label>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      data-theme="bumblebee"
    >
      <nav
        class="navbar bg-primary"
      >
        <div
          class="flex-1"
        >
          <a
            class="btn btn-ghost normal-case text-xl"
            href="/"
          >
            useGeolocation
          </a>
        </div>
        <div
          class="flex-none"
        >
          <ul
            class="menu menu-horizontal p-0"
          >
            <li>
              <a
                href="https://github.com/streamich/react-use/blob/master/docs/useGeolocation.md"
                target="__blank"
              >
                Docs
              </a>
            </li>
          </ul>
        </div>
      </nav>
      <div
        class="hero min-h-screen bg-base-200"
      >
        <div
          class="hero-content text-center"
        >
          <div
            class="max-w-md"
          >
            <h1
              class="text-5xl font-bold"
            >
              useGeolocation
            </h1>
            <p
              class="py-6"
            >
              A React hook that grabs a user's coordinates.
            </p>
            <button
              class="btn btn-primary btn-wide"
            >
              Get Location
            </button>
            <div
              class="w-1/2 mx-auto mt-4"
            >
              <label
                class="label cursor-pointer"
              >
                <span
                  class="label-text"
                >
                  High accuracy
                </span>
                <input
                  class="toggle toggle-primary"
                  type="checkbox"
                />
              </label>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
